src/openfl/utils/Assets.hx

```diff
@@ -162,7 +167,23 @@ class Assets
 			#if flash
 			var bitmapData = image.src;
 			#else
-			var bitmapData = BitmapData.fromImage(image);
+			var bitmapData:BitmapData = BitmapData.fromImage(image);
+			#if (!macro && desktop)
+			if (pushToGPU && Assets.allowGPU)
+			{
+				bitmapData.lock();
+				if (bitmapData.__texture == null)
+				{
+					bitmapData.image.premultiplied = true;
+					bitmapData.getTexture(flixel.FlxG.stage.context3D);
```

Comment:
Yeah don't reference Flixel here ideally.

Reply:
You can use `openfl.Lib.current.stage.context3D` here instead since `flixel.FlxG.stage.context3D` is just a shortcut to that.